package.json

```diff
@@ -4,9 +4,11 @@
   "description": "",
   "main": "main.js",
   "scripts": {
-    "start": "NODE_ENV=development node ./server/server.js",
-    "build": "NODE_ENV=production node_modules/.bin/webpack --config ./configWebpack/prod.js --bail --progress",
-    "serve": "NODE_ENV=production node ./server/server.js"
+    "start": "cross-env NODE_ENV=development node ./server/server.js",
+    "build": "cross-env NODE_ENV=production node ./server/generateStatistics.js &&  cross-env NODE_ENV=production  node_modules/.bin/webpack --config ./configWebpack/prod.js --bail --progress",
+    "serve": "cross-env NODE_ENV=production node ./server/server.js",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
```

Comment:
Are you not storing the build in `\docs`?, if so, this needs to be `gh-pages -d docs`

Reply:
I think this still needs to be changed?